demisto_client/__init__.py

```diff
@@ -61,6 +63,11 @@ def configure(base_url=None, api_key=None, verify_ssl=None, proxy=None, username
         username = os.getenv('DEMISTO_USERNAME')
     if password is None:
         password = os.getenv('DEMISTO_PASSWORD')
+    if additional_headers is None:
+        if headers := os.getenv('DEMISTO_HTTP_HEADERS'):
+            additional_headers = dict(header.split('=') for header in headers.split(','))
```

Comment:
let's delight the customers by adding some try-except block when an invalid format is used

Reply:
done